### PR TITLE
Change DBus property tests to pass DBusException

### LINKED ISSUE #129: Improve unittest_set_param and pool_set_param methods
```pool_set_param``` catches any D-Bus exception and turns it into a boolean. This causes problems if the D-Bus call fails, making it hard to diagnose. We should probably allow the exception to be propagated to the caller and deal with it there. The only caller is ```unittest_set_param``` at this point.

Also, we need to change their names to ```unittest_set_property``` and ```pool_set_property```.

### PR DESCRIPTION
**Pull request by bgurney-rh**
Closes #129

### CODE REVIEW

File: testlib/dbus.py [hunk 1]
```diff
@@ -407,17 +407,12 @@ def pool_set_param(pool_path, param_iface, dbus_param, dbus_value):
...
-                param_iface,
-                dbus_param,
-                dbus_value,
-                timeout=StratisDbus._TIMEOUT,
-            )
-        except dbus.exceptions.DBusException:
-            return False
-
-        return True
+        return iface.Set(
```
Comment (mulkieran) [inline testlib/dbus.py:410]: Document what the method returns in the docs. You could also document that it raise dbus.exceptions.DBusException.

File: stratisd_cert.py [hunk 2]
```diff
@@ -1001,7 +1012,7 @@ def test_get_report(self):
         """
         Test getting a valid and invalid report.
         """
-        (result, return_code, _) = StratisDbus.get_report("stopped_pools")
+        (result, return_code, _) = StratisDbus.get_report("errored_pool_report")
```
Comment (mulkieran) [inline stratisd_cert.py:1015]: This is just a rebase problem....

**Review by mulkieran — CHANGES_REQUESTED** [review]
A couple of small requests, otherwise looks good.

File: stratisd_cert.py (hunk 2 above)
Comment (bgurney-rh) [reply stratisd_cert.py:1015]: Actually, no; this is the commit `Revert "stratisd: change test_get_report valid test to stopped_pools"` in this PR's branch, which I would need, to ensure that this test passes.  Is it okay if I add this commit?

Comment (mulkieran) [reply stratisd_cert.py:1015]: @bgurney-rh Go ahead.

**Pull request closed without merging**